config/packages/messenger.yaml

```diff
@@ -9,22 +9,31 @@ framework:
             async:
                 dsn: "%env(MESSENGER_TRANSPORT_DSN)%"
                 retry_strategy:
-                    max_retries: 0
-                    delay: 1800000
-                    multiplier: 2
-                    max_delay: 0
+                    max_retries: 5
+                    delay: 600000
+                    multiplier: 10
+                    max_delay: 604800000
                 serializer: messenger.transport.symfony_serializer
             async_ap:
                 dsn: "%env(MESSENGER_TRANSPORT_DSN)%"
                 retry_strategy:
-                    max_retries: 0
-                    delay: 1800000
-                    multiplier: 2
-                    max_delay: 0
+                    max_retries: 5
```

Comment:
I approved as you notice. But I'm also fine with max 3 retries personally. I leave it up to you.

Reply:
I'd keep it at 5 I think. If we only do 3 retries there is only a ~1 day delay until we decide this message is failed. I'd be fine with 4, so we a little more than a week delay in total instead of a little over two weeks